dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/AndroidTracer.kt

```diff
@@ -163,6 +167,9 @@ class AndroidTracer internal constructor(
         fun logErrorMessage(span: Span, message: String) {
             val fieldsMap = mapOf(Fields.MESSAGE to message)
             span.log(fieldsMap)
+            if (span is DDSpan) {
+                span.setError(true)
```

Comment:
You do not need this....it is already done in the AndroidLogsHandler by calling : `span.setErrorMeta` which inside calls `span.setError(true)`